### DISCUSSION TITLE
Hasil Submit Capstone Tidak Muncul di Leaderboard

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#159] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/95233522/175764049-809e0200-7fb9-415c-a255-d08600f3876d.JPG)

Ini hasil submit enggak muncul di leaderboard, apa karena ada error itu?

Saya upload lebih dari 3 kali juga tetap bisa diupload, berarti hasil upload sebelumnya nggak keterima? 

Padahal sudah muncul hasilnya accuracy saya 51%. Tapi diupload berkali-kali nggak keterima sistem. 
![aaa](https://user-images.githubusercontent.com/95233522/175764391-70bd3bb6-45ce-4833-a1be-2b89e8fdaf07.JPG)

**Comment by Hertzspr** [comment]
Apakah karena masih merah jadi nggak kerekam yah?

**Comment by kevbow** [comment, marked as answer]
Halo @Hertzspr, 

Error tersebut tidak usah dihiraukan, silakan lihat bagian model performance saja untuk difoto hasilnya.
Demikian dan terima kasih.

**Reply by Hertzspr** [reply]
Ok, terima kasih.
